ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py

```diff
@@ -2552,4 +2552,4 @@ def test_tensorflow_conj(
         fn_tree=fn_tree,
         on_device=on_device,
         x=x[0],
-    )
```

Comment:
Also remove this change from the branch and the PR

Reply:
ok